tutorials/cloudbuild-angular-universal/index.md

```diff
@@ -0,0 +1,301 @@
+```
+git add cloudbuild.yaml && git commit -m "add cloudbuild.yaml"
+```
+### Create a Build Trigger that will build, test and deploy your application to the Cloud CDN
+You can create a trigger on the [build triggers page](https://console.cloud.google.com/cloud-build/triggers) of the GCP Console by following these steps:
+
+1. Click **Create Trigger** OR **Add Trigger** button
+1. Select **Cloud Source Repository** Click **Continue**.
+1. Select ```tour-of-heroes-universal``` and click **Continue**.
+1. Enter ```angular-universal-tour``` for **Name.**
+1. Under **Trigger type** select "Tag".
+1. Under **Build configuration** select ```cloudbuild.yaml```
+1. Under **Substitution variables** press **+ Add item**
+1. In the **Variable** field enter```_ANGULAR_APP_BUCKET_PATH``` and in **Value** enter ```gs://[PROJECT]-angular-app``` where ```[PROJECT]``` is the name of your project
+1. Click **Create Trigger**
+
+### Add your tour-of-heros cloud source repository as a remote repository with the name 'google'
+
+1.  (**OPTIONAL**: Only if not running in Cloud Shell) Set up your google credentials for git.
+```
+gcloud init && git config --global credential.https://source.developers.google.com.helper gcloud.sh
+```
+
+2.  Add the google cloud repo as a remote.
+```
+git remote add google \
+https://source.developers.google.com/p/$PROJECT/r/tour-of-heros-universal
+```
+
+## Run the Build Trigger and Deploy the Application
+
+1.  Tag the repository.
+```
+git tag v0.0
+```
+
+2.  Push the repository to google.
+```
+git push google master && git push google --tags
+```
+
+### Once the build and deploy finishes check that the website is deployed
+1.  Open up the [Cloud Build](https://console.cloud.google.com/cloud-build) console to show the build progress.
+2.  Find the build that is in progress and click the link to view its progress.
+3.  Once the build finishes, find the IP address of the load balancer you created above:
+
+```
+gcloud compute addresses list  --filter="name=angular-app-ip" --format="value(address)"
+```
+
+4. Point your browser at ```http://[ANGULAR_APP_IP]```, replacing ```[ANGULAR_APP_IP]``` with the IP address retrieved above.
+
+### Cleanup
+1. Delete the load balancer.
+
+```
+gcloud compute forwarding-rules delete http-content-rule --global --quiet
+
+gcloud compute target-http-proxies delete http-lb-proxy --quiet
+
+gcloud compute url-maps delete web-map  --quiet
+
+gcloud compute addresses delete angular-app-ip --global --quiet
+```
+
+2. Delete the cloud storage bucket.
+```
+gcloud compute backend-buckets delete $PROJECT-angular-app-backend --quiet
+
+gsutil rm gs://$PROJECT-angular-app/**
+gsutil rb gs://$PROJECT-angular-app
+```
+3. Delete the Build Trigger
+
+    1. Navigate to [build triggers page](https://console.cloud.google.com/cloud-build/triggers) of the GCP Console 
+    1. On the line of the build trigger "tour-of-heroes-universal" select the menu ![trigger button](trigger-button.png)
+    1. Select "Delete"
```

Comment:
As you have done with most UI strings, use bold formatting, not quotation marks.